packages/app/src/app/pages/Sandbox/SearchDependencies/Dependency.tsx

```diff
@@ -98,10 +117,42 @@ export const Dependency = ({ dependency }: { dependency: DependencyType }) => {
   const getTagName = (tags: DependencyType['tags'], version: string) =>
     Object.keys(tags).find(key => tags[key] === version);
 
+  const onVersionSelect = e => {
+    const {
+      target: { value },
+    } = e;
```

Comment:
```suggestion
  const onVersionSelect = ({ target: { value } }) => {
```

Reply:
Just a question, do we follow a particular code style for this? Like destructuring in param or destructing separately from a variable ?